### PR TITLE
chore: initial setup backend

### PR DESCRIPTION
**Pull request by hamsof**

### CODE REVIEW

File: backend/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  root: true,
+  env: {
+    node: true,
+    jest: true,
+  },
+  ignorePatterns: ['.eslintrc.js'],
+  rules: {
+    '@typescript-eslint/interface-name-prefix': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
```
Comment (Shamsa-H) [inline backend/.eslintrc.js:22]: why off? Shouldn't we keep it on to maintain a clean code base

File: backend/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    'plugin:prettier/recommended',
+  ],
+  root: true,
+  env: {
+    node: true,
+    jest: true,
+  },
+  ignorePatterns: ['.eslintrc.js'],
+  rules: {
+    '@typescript-eslint/interface-name-prefix': 'off',
```
Comment (Shamsa-Hafeez1) [inline backend/.eslintrc.js]: why off? Shouldn't we keep it on to maintain a clean code base

Comment (hamsof) [reply backend/.eslintrc.js]: So I am using SWC compiler to compile TS and it has made these changes I think I will into this

Comment (hamsof) [reply backend/.eslintrc.js]: other than this you can approve if its ok

**Review by Shamsa-Hafeez1 — DISMISSED** [review]

**Review by Shamsa-H — APPROVED** [review]

**Pull request merged**